### PR TITLE
ENG 2405 Fix bug where log and stack traces divs don't have backgrounds

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
Add the colors back to the logs :)

## Related issue number (if any)
ENG 2405

## Demo
Error Log
Empty Log
StdErr Log
StdOut Log

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [N/A] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/LogViewer/index.tsx [hunk 1]
```diff
@@ -15,7 +16,9 @@ const LogViewer: React.FC<Props> = ({ logs, err }) => {
 
   const [selectedTab, setSelectedTab] = useState(0);
   const emptyElement = (
-    <Box sx={{ p: 2, backgroundColor: 'gray.100' }}>Nothing to see here!</Box>
+    <Box sx={{ p: 2, backgroundColor: theme.palette.gray[100] }}>
```
Comment (agiron123) [inline src/ui/common/src/components/LogViewer/index.tsx:19]: Seems our sx prop isn't working with colors.

I'm not entirely sure what's going on there, but could be something we reach out to the MUI team for. This issue started appearing a couple weeks back when the frontend build did not work in dev mode.

Comment (agiron123) [reply src/ui/common/src/components/LogViewer/index.tsx:19]: Can you try importing useTheme from MUI and logging the color palette to make sure that the gray[100] key is there?

Could be an issue of spelling of gray vs grey.

```
import { useTheme } from '@mui/material/styles';
const muiTheme = useTheme();
console.log('theme from context: ', theme)
```

**Review by agiron123 — APPROVED** [review]
Code looks good, but left some comments related to this issue that we should take a look into.

File: src/ui/common/src/components/LogViewer/index.tsx (hunk 1 above)
Comment (eunice-chan) [reply src/ui/common/src/components/LogViewer/index.tsx:19]: image

**Pull request merged**